net/http/civetweb/civetweb.c

```diff
@@ -2699,7 +2706,7 @@ mg_set_thread_name(const char *name)
     && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 12)))
 	/* pthread_setname_np first appeared in glibc in version 2.12*/
 	(void)pthread_setname_np(pthread_self(), threadName);
-#elif defined(__linux__)
+#elif defined(__linux__) || defined(__clang__)
```

Comment:
One is platform, the other compiler. Those are not compatible, e.g. `__clang__` might be defined on Windows. Why do you need this?

Reply:
Seems to be, this was wrong - there is already compilation error on mac. 
One should try to use pthread_setname_np